rows.go

```diff
@@ -101,6 +102,9 @@ func (rows *Rows) Next() bool {
 					rows.curRow = rowNum
 				}
 				rows.token = token
+				if styleID, _ := attrValToInt("s", xmlElement.Attr); styleID != 0 {
```

Comment:
I agree with the return style ID of the row as is, without checking if which is less than `style.CellXfs.Count`, but I suggest that make `styleID` in uint data type and add the simple guard like this:


define a new exported constant in the `xmlDrawing.go`:

```go
const MaxCellStyles = 64000
```

```go
if styleID, _ := attrValToInt("s", xmlElement.Attr); 0 <= styleID && styleID < MaxCellStyles {
	rows.styleID = uint(styleID)
}
```

Reply:
Actually, i was thinking, does it make more sense to return a `RowOpts` instead of just the StyleID?